go/cmd/vtctldclient/command/tablets.go

```diff
@@ -99,7 +99,7 @@ Note: hook names may not contain slash (/) characters.
 	}
 	// GetTablets makes a GetTablets gRPC call to a vtctld.
 	GetTablets = &cobra.Command{
-		Use:   "GetTablets [--strict] [{--cell $c1 [--cell $c2 ...], --keyspace $ks [--shard $shard], --tablet-alias $alias}]",
+		Use:   "GetTablets [--strict] [{--cell $c1 [--cell $c2 ...], [--tablet-type $t1] $ks [--shard $shard], --tablet-alias $alias}]",
```

Comment:
just realized this is wrong, `--keyspace` is still a flag argument if i'm reading the rest right